crypto/asn1/a_time.c

```diff
@@ -208,6 +208,10 @@ int ASN1_TIME_diff(int *out_days, int *out_seconds, const ASN1_TIME *from,
 // slightly different than the many other copies of X.509 time validation
 // sprinkled through the codebase. The custom checks in X509_cmp_time meant that
 // it did not allow four digit timezone offsets in UTC times.
+int ASN1_TIME_to_tm(const ASN1_TIME *s, struct tm *tm) {
+  return asn1_time_to_tm(tm, s, /*allow_timezone_offset=*/0);
+}
```

Comment:
Should a `tm == NULL` check be occurring here or in [`asn1_time_to_tm`](https://github.com/aws/aws-lc/blob/7923ea46e920b10ba09241ae7ff50692ae476feb/crypto/asn1/a_time.c#L173)?  I don't see a check on that value for at least one of the paths the logic can take to reach the `memset` at the start of [`OPENSSL_posix_to_tm`](https://github.com/aws/aws-lc/blob/7923ea46e920b10ba09241ae7ff50692ae476feb/crypto/asn1/posix_time.c#LL148C1-L148C1).